src/Providers/Trustpilot/Request/AccessTokenRequest.php

```diff
@@ -0,0 +1,76 @@
+<?php
+
+namespace PodPoint\Reviews\Providers\Trustpilot\Request;
+
+use GuzzleHttp\Psr7\Request;
+use PodPoint\Reviews\AccessToken;
+use PodPoint\Reviews\Request\BaseRequest;
+
+/**
+ * Class AccessTokenRequest
+ * @package PodPoint\Reviews\Providers\Trustpilot\Request
+ */
+class AccessTokenRequest extends BaseRequest
+{
+    const CLIENT_ID = 'client_id';
+    const CLIENT_SECRET = 'client_secret';
+    const USERNAME = 'username';
+    const PASSWORD = 'password';
+
+    public function requiredFields(): array
+    {
+        return [
+            self::CLIENT_ID,
+            self::CLIENT_SECRET,
+            self::USERNAME,
+            self::PASSWORD,
+        ];
+    }
+
+    /**
+     * Builds the request.
+     *
+     * @return Request
+     */
+    public function getRequest(): Request
+    {
+        $key = base64_encode("{$this->getOption(self::CLIENT_ID)}:{$this->getOption(self::CLIENT_SECRET)}");
+
+        $method = 'POST';
+        $uri = '/v1/oauth/oauth-business-users-for-applications/accesstoken';
```

Comment:
move $uri up as class constant

Reply:
double check that test checks for content type because it's crucial for functionality